frontend/src/libs/helpers/get-format-time.ts

```diff
@@ -0,0 +1,17 @@
+const SECONDS_NUMBER = 60;
+const SECONDS = 10;
+
+const formatTime = (time: number): string => {
+  if (time && !Number.isNaN(time)) {
+    const minutes = Math.floor(time / SECONDS_NUMBER);
+    const formatMinutes = minutes < SECONDS ? `0${minutes}` : `${minutes}`;
```

Comment:
I think you can use this suggestion and remove the variable `SECONDS`, cause the naming is quite confusing since you used It to check if the number is a double-digit one
```suggestion
    const formatMinutes = minutes.toString().padStart(2, '0');
```

Reply:
@Nanaier, that's quite a cool tip)